dist/server/setdist.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+export APP_PORT=6379
+export JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64
+export RELEASE=carrot-1.0.0
+export DISTRIBUTION=${RELEASE}.tar.gz
+export HEALTH_MONITOR=http://localhost:${APP_PORT}/ping
```

Comment:
We do not do HEALTH_MONITOR? Please remove

Reply:
removed